examples/hdr/rs-hdr.cpp

```diff
@@ -103,12 +120,51 @@ int main(int argc, char * argv[]) try
     // setting the required sequence ID to be shown
     spliting_filter.set_option(RS2_OPTION_SEQUENCE_ID, 2);
 
+    rs2::frameset data;
+
     // flag used to see the original stream or the merged one
-    bool true_for_merge_false_for_split = true;
     int frames_without_hdr_metadata_params = 0;
-    while (app) // Application still alive?
+
+    //create a map to hold frames with their properties
+    map_of_frames_and_tiles_properties frames_map;
```

Comment:
Refactor all the auxillary widgets into a class in order to hide all the low-level mechanics, i.e. instead of 

```cpp
        exposure_slider_2.show();
        exposure_slider_1.show();
        gain_slider_2.show();
        gain_slider_1.show();
```
 iit will use
```cpp
hdr_widgets_class.render_sliders();
```

instead of 
```cpp
app.show(frames_map);
```
use
```cpp
app.show(hdr_widgets_class.get_frames_map());
```
the same applies to the initialization that will be encapsulated and not seen in main function